### PR TITLE
example : MultiDocument Agentic RAG with Function tools

### PR DESCRIPTION
**Pull request by viplismism**
- [ ] **Add Colab Link to Blog:** After merging the example, ensure the Colab link is included in the blog post before it’s published.
- [ ] **Update `README.md`:** Once the blog is published, verify that the blog link is correctly added to the `README.md` for this example.

### CODE REVIEW

**Comment by vipul-maheshwari** [comment]
@AyushExel

**Review by AyushExel — APPROVED** [review]

File: README.md
Comment (PrashantDixit0) [inline README.md:1]: 1. colab notebook link is broken (wrong folder name)
2. Please verify the link of blog when it is published

File: examples/multi-document-agentic-rag/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Multidocument Agentic RAG
+
+![alt text](../../assets/multidocument-agentic-rag.png)
```
Comment (PrashantDixit0) [inline examples/multi-document-agentic-rag/README.md:3]: image link is broken

File: examples/multi-document-agentic-rag/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+This example provides a comprehensive guide on creating a Multi-Document Agentic RAG leveraging the power of Embeddings and VectorDB. We'll explore how we can use Reasoning + Acting (ReAct) strategy to harness our RAG setup and make it more intelligent using different tools.
+
+Colab walkthrough - <a href="https://colab.research.google.com/github/lancedb/vectordb-recipes/blob/main/examples/Multidocument-Agentic-RAG/main.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
```
Comment (PrashantDixit0) [inline examples/multi-document-agentic-rag/README.md:7]: Colab link will break due to wrong folder name

File: examples/multi-document-agentic-rag/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Multidocument Agentic RAG
+
+![alt text](../../assets/multidocument-agentic-rag.png)
+
+This example provides a comprehensive guide on creating a Multi-Document Agentic RAG leveraging the power of Embeddings and VectorDB. We'll explore how we can use Reasoning + Acting (ReAct) strategy to harness our RAG setup and make it more intelligent using different tools.
+
+Colab walkthrough - <a href="https://colab.research.google.com/github/lancedb/vectordb-recipes/blob/main/examples/Multidocument-Agentic-RAG/main.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> 
+
+[Read the Blog Post](https://blog.lancedb.com/multi-document-agentic-rag/)
```
Comment (PrashantDixit0) [inline examples/multi-document-agentic-rag/README.md:9]: verify it once blog is published

File: examples/multi-document-agentic-rag/main.ipynb
Comment (PrashantDixit0) [inline examples/multi-document-agentic-rag/main.ipynb:1]: Please add instructions and explanations at each step. Add headings to break down code into different sections

File: examples/multi-document-agentic-rag/README.md (hunk 1 above)
Comment (vipul-maheshwari) [reply examples/multi-document-agentic-rag/README.md:3]: Fixed

File: examples/multi-document-agentic-rag/README.md (hunk 2 above)
Comment (vipul-maheshwari) [reply examples/multi-document-agentic-rag/README.md:7]: Fixed

File: examples/multi-document-agentic-rag/main.ipynb
Comment (vipul-maheshwari) [reply examples/multi-document-agentic-rag/main.ipynb:1]: Headings are already there! Adding some explanation points

Comment (vipul-maheshwari) [reply examples/multi-document-agentic-rag/main.ipynb:1]: Done

File: examples/multi-document-agentic-rag/README.md (hunk 3 above)
Comment (vipul-maheshwari) [reply examples/multi-document-agentic-rag/README.md:9]: Will do this after blog is published!

File: README.md
Comment (vipul-maheshwari) [reply README.md:1]: Fixed

**Pull request merged**